### ISSUE TITLE
GTK: Tray icon context menu entry `View script error` broken

### ISSUE THREAD

**Issue by luziferius** [autokey/autokey#222]
The bug was mentioned in the discussion of issue #221.

## Reproducibility:

Always

## Summary

Tray icon context menu entry `View script error` broken, if the main window is not opened

## Steps to Reproduce (if applicable)

- Make sure the main window is closed
- Execute a script, causing an error
- Right click the AutoKey tray icon
- Click on `View script error`

## Expected Results
A dialogue window opens and shows the Python backtrace of the last failed script.

## Actual Results

- Instead, nothing happens. There is an exception thrown on the standard output.

```python3
Traceback (most recent call last):
  File '~/autokey/gtkui/notifier.py", line 140, in on_show_error
    self.app.show_script_error(self.app.configWindow.ui)
AttributeError: 'NoneType' object has no attribute 'ui'
```

## Version

AutoKey version: 0.95.4, git master, git develop

Used GUI (Gtk, Qt, or both): autokey-gtk

Installed via: local git

Distro: Kubuntu 18.04

## Notes

The UI is destroyed when closed and the tray icon tries to access the already destroyed and deleted UI classes. Unless the GTK GUI is properly refactored to not do this, the entry should probably create a new GUI instance to be able to show its message.

**Comment by ineuw** [comment]
Using autokey-gtk 0.95.1 in Linux Mint 19.1 
The tray menu icon for errors is still not working.
Python 2.7.15rc1 (default, Nov 12 2018, 14:31:15) 
Python 3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0] on linux

**Comment by luziferius** [comment]
@ineuw The just released version 0.95.7 includes a fix for this. Due to how the GTK GUI works, a proper fix needs a major code overhaul, so I just added a workaround, until that overhaul is adressed.

Now the GTK GUI will open the main window when using this context menu entry, because the entry requires some components of the main window, which are unavailable when the main window is closed.
